src/or/connection.c

```diff
@@ -2658,16 +2687,27 @@ retry_listener_ports(smartlist_t *old_conns,
           break;
         }
       } else {
-        int port_matches;
-        if (wanted->port == CFG_AUTO_PORT) {
-          port_matches = 1;
-        } else {
-          port_matches = (wanted->port == conn->port);
-        }
+        int port_matches_exact = (wanted->port == conn->port);
```

Comment:
Let's make these variables const, and document what we mean by them.

Reply:
Did so in 118cf220f6c81f54beeff3d31ecf0d2dfb3e83fc.